src/Bridges/debug.jl

```diff
@@ -39,7 +48,7 @@ function print_node_info(
 )
     print(io, " ")
     print_node(io, b, node)
-    d = _dist(b.graph, node)
+    d = _cost(b.graph, node)
```

Comment:
I'd like to revert all of these naming changes. Let's keep the minimal set of changes for this PR. `_dist` and `_cost` are synonymous anyway.